examples/client/Locomotion/src/pages/ActiveRide/RideDrawer/RideOptions/RideButtons/index.tsx

```diff
@@ -73,7 +73,7 @@ const RideButtons = ({
         <DatePicker
           textColor={theme.textColor}
           open={isDatePickerOpen}
-          date={moment(ride?.scheduledTo).add(ride?.scheduledTo ? 0 : 1, 'hours').toDate()}
+          date={moment(ride?.scheduledTo || undefined).add(ride?.scheduledTo ? 0 : 1, 'hours').toDate()}
```

Comment:
ride?.scheduledTo ? moment(ride.scheduledTo).toDate() : moment().add(1, 'hours').toDate()

Reply:
why do we want to put 1 hour from now by default? isnt is it should be a setting or something?